src/firefly/java/edu/caltech/ipac/util/FileUtil.java

```diff
@@ -25,264 +34,279 @@
+
+    public final static String jpg = "jpg";
+    public final static String png = "png";
+    public final static String bmp = "bmp";
     public final static String FITS = "fits";
-    public final static String GZ   = "gz";
-    public final static String PS   = "ps";
-    public final static String PDF  = "pdf";
-    public final static String HTML = "html";
-    public final static String AOR  = "aor";
-    public final static String TXT  = "txt";
-    public final static String TGT  = "tgt";
-    public final static String ZIP  = "zip";
-    public final static String TBL  = "tbl";
-    public final static String NL   = "nl";
-    public final static String JAR   = "jar";
-    public final static String CSH   = "csh";
-    public final static String SH   = "sh";
-    public final static String PL   = "pl";
-    public final static String SO   = "so";
-    public final static String REG  = "reg";
-
-    public static final long MEG          = 1048576;
-    public static final long GIG          = 1048576 * 1024;
-    public static final long MEG_TENTH    = MEG / 10;
-    public static final long GIG_HUNDREDTH= GIG / 100;
-    public static final long K            = 1024;
-
-    private static String[] CHARSETS_TO_BE_TESTED = {"ISO-8859-1","windows-1252", "windows-1253","UTF-8", "UTF-16",};
+    public final static String GZ = "gz";
+    public final static String PDF = "pdf";
+    public final static String JAR = "jar";
+
+    public static final long MEG = 1048576;
+    public static final long GIG = 1048576 * 1024;
+    public static final long MEG_TENTH = MEG / 10;
+    public static final long GIG_HUNDREDTH = GIG / 100;
+    public static final long K = 1024;
+
+    private static String[] CHARSETS_TO_BE_TESTED = {"ISO-8859-1", "windows-1252", "windows-1253", "UTF-8", "UTF-16",};
     public static final int BUFFER_SIZE = (int) (32 * FileUtil.K);
 
-    public enum ConvertTo {MEG,GIG,K}
+    public enum ConvertTo {MEG, GIG, K}
 
-    private static String hostNameReturn= null;
+    private static String hostNameReturn = null;
 
-  private final static String  ADD_START= "---";
+    private final static String ADD_START = "---";
 
     /**
      * Get the extension of a filename.
-     * @param  s a file name such as <code>a.dat</code>
+     *
+     * @param s a file name such as <code>a.dat</code>
      * @return String the extension of the file.
-     *                A null is returned if there is no extension; 
-     *                
+     * A null is returned if there is no extension;
      */
-  public static String getExtension(String s) 
-  {
-    String ext = "";
-    int i = s.lastIndexOf('.');
-    if (i > 0 &&  i < s.length() - 1) 
-    {
-      ext = s.substring(i+1).toLowerCase();
+    public static String getExtension(String s) {
+        String ext = "";
+        int i = s.lastIndexOf('.');
+        if (i > 0 && i < s.length() - 1) {
+            ext = s.substring(i + 1).toLowerCase();
+        }
+        return ext;
```

Comment:
Did you mean to reformat all of `FileUtil.java`?

Reply:
Yes, the file was poorly formatted, and I had intended to clean up FileUtil. However, based on your suggestion to move my logic elsewhere, I’ll go ahead and revert the entire file.